spec/models/book_bag_spec.rb

```diff
@@ -29,4 +29,13 @@
       expect(described_class.active).to match_array actives
     end
   end
+
+  describe '.by_global_id' do
+    it 'sorts bags by global ID' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.